priv/repo/migrations/20190714140153_add_points_to_cards.exs

```diff
@@ -0,0 +1,9 @@
+defmodule PointingParty.Repo.Migrations.AddPointsToCards do
+  use Ecto.Migration
+
+  def change do
+    alter table("cards") do
+      add :points, :integer, default: 0
+    end
```

Comment:
👍 Will the `Calculate Points` link eventually save the `points` value to the database?

Reply:
Yep! I was thinking we can have this button hidden and then show up after each participant has voted. Or something like that.